platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -56,6 +65,25 @@ SECTIONS
         __vectors_end__ = .;
     } > FLASH
 
+#ifndef TFM_MULTI_CORE_TOPOLOGY
+    /*
+     * Place the CMSE Veneers (containing the SG instruction) after the code, in a
+     * separate 32 bytes aligned region so that the SAU can programmed to just set
+     * this region as Non-Secure Callable.
+     */
+    .gnu.sgstubs : ALIGN(32)
+    {
+        SG_ALIGN;
+        Load$$LR$$LR_VENEER$$Base = ABSOLUTE(.);
+        *(.gnu.sgstubs*)
+    } > VENEERS AT> VENEERS
+    SG_END_ALIGN;
+    Load$$LR$$LR_VENEER$$Limit = . - 1;
+
+    ASSERT ((Load$$LR$$LR_VENEER$$Limit - Load$$LR$$LR_VENEER$$Base)
+            <= CMSE_VENEER_REGION_SIZE, "Veneer region overflowed")
```

Comment:
After the definition of `Load$$LR$$LR_VENEER$$Limit` has been changed, this assertion should be updated to use `<` instead of `<=`.

Reply:
Good catch!

Reply:
Fixed.